fs/walk.py

```diff
@@ -198,10 +215,19 @@ def _iter_walk(
     def _check_open_dir(self, fs, path, info):
         # type: (FS, Text, Info) -> bool
         """Check if a directory should be considered in the walk."""
+        full_path = ("" if path == "/" else path) + "/" + info.name
```

Comment:
Is this a clunky way of doing https://docs.pyfilesystem.org/en/latest/reference/path.html#fs.path.combine ?